client/src/components/AuditBoard/__snapshots__/AuditBoardView.test.tsx.snap

```diff
@@ -61,19 +61,19 @@ exports[`AuditBoardView ballot interaction renders ballot route 1`] = `
         class="sc-bwzfXH ijTqJq"
       >
         <div
-          class="sc-bMvGRv eoqrva"
+          class="sc-CtfFt dyWieC"
```

Comment:
I didn't make any change related to this file, and I see that @jonahkagan also had a similar change to this snapshot in [this PR that didn't touch the associated file](https://github.com/votingworks/arlo/pull/2000/files#diff-77684cd11a1fe7b0251766ae865668179aa663969064792892f030cf467c0f98R10), so I assume this is safe.

Reply:
@arsalansufi any insight on what might be triggering this change? I see Jonah's [commit message](https://github.com/votingworks/arlo/pull/2000/commits/dc789f1c931b89c74c65a430b4e968dfe13678a9) referenced styled components and all the classes changed are prefixed with `sc`, so maybe this is some behavior we have seen before

Reply:
I think when you add a new styled component (or update one, I can't quite remember) it regenerates all the class names, which are included in the snapshots. We found a [plugin](https://github.com/styled-components/jest-styled-components) that fixes this issue and added it to VxSuite, but haven't gotten around to doing it in Arlo.

Reply:
Ah okay, thanks for the explanation